docs/theming.md

```diff
@@ -50,7 +50,8 @@ through Material Theming.
 
 ```swift
 import MaterialComponents.MaterialButtons
-import MaterialComponentsBeta.MaterialButtons_Theming
+import MaterialComponents.MaterialButtons_Theming
+import MaterialComponents.MaterialContainerScheme
```

Comment:
Is this import required?

Reply:
I feel that if we are going to have a document on theming and import the things required for the component we should also import the needs for theming. What do you think @jverkoey ?

Reply:
My thinking was that importing the component’s theming extension should be sufficient to theme a component, and that extra imports may add to perceived overhead of theming.

I don’t feel particularly strongly on this regard though. My reasoning basically comes down to “if we can get away with less then that’s less the client needs to learn”.

Reply:
Will LGTM either way you decide here Cody :)

Reply:
I thought it was required but I tested this in a _play_ app and we only need the two imports "Buttons" and "Buttons_Theming". I have removed the container scheme import.

Reply:
I would vote for strict dependency imports in general. I think it's best to import whatever the code depends on to avoid relying on transitive imports.

Reply:
@romoore that's fair, but for the purposes of this documentation I want to focus on conciseness rather than correctness; I'd like to leave it to the reader to decide how they want to enforce import correctness.

Reply:
In that case, can we omit `Buttons_Theming` since it should be pulling-in `Buttons`?

Reply:
We could; but we're pulling in both in this case.